stdlib/proposals/ai/evaluation_for_ai_agents_in_ballerina.md

```diff
@@ -0,0 +1,321 @@
+# Evaluation for AI Agents in Ballerina
+
+- Authors
+  - @MohamedSabthar
+- Reviewed by
+    - @shafreenAnfar
+- Created date
+    - 2025-11-21
+- Issue
+    - [1402](https://github.com/ballerina-platform/ballerina-spec/issues/1402)
+- State
+    - Submitted
+
+## Summary
+
+This proposal introduces an observability-free, test-driven evaluation method for AI Agents in Ballerina. Unlike frameworks such as Arize Phoenix or Langfuse that rely heavily on observability traces, this approach focuses on input–output evaluation and behavioral assertions, allowing validation of agent behavior without requiring any observability infrastructure.
```

Comment:
Given that this is the language spec, do we need the comparison with other frameworks by name?